### PR TITLE
Radioactive HyperPacman Generator

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Added ``RadiationSource`` to new hyper pacman generator, works only when they are running, 0.1 rad per 1 kW.

Can be found in expos on that one tiny rad safe room.

Also small JrPacman buff

## Why / Balance
Added a new radiation mini system for portable generators and added it to a new generator to be mapped later. 

## How to test
Spawn the new generator and let it run

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

## Breaking changes
N/A

:cl: dvir01
- add: You can now find a new Bananium power generator on expos, but make sure you can protect yourself when running it.

### CODE REVIEW

**Review by whatston3 — DISMISSED** [review]
Looks good, works well.

**Comment by Cheackraze** [comment]
Lots of problems with radiation conceptually, including being absolutely garbage gameplay-wise of having little to no counter play, having *no* discoverability, and providing hacky workarounds to core gameplay loops, trivializing several key elements of gameplay.

I dont think this is a good change

**Comment by dvir001** [comment]
This PR has been fully reworked, moved into new generator.

**Comment by whatston3** [comment]
Can't get the game to run on head of branch in local tests, dismissing my approval for now.

File: Resources/Maps/_NF/Dungeon/cave_factory.yml [hunk 1]
```diff
@@ -6758,6 +6783,21 @@ entities:
...
+  - uid: 1944
+    components:
+    - type: Transform
+      pos: 32.5,19.5
+      parent: 1
+  - uid: 1945
+    components:
+    - type: Transform
+      pos: 31.5,19.5
+      parent: 1
```
Comment (whatston3) [inline Resources/Maps/_NF/Dungeon/cave_factory.yml:6800]: Game doesn't seem to like these entities, their IDs are already in use for engineering loot spawn.

Comment (dvir001) [reply Resources/Maps/_NF/Dungeon/cave_factory.yml:6800]: Might have fucked something with the grid save, I can fix in few hours.

**Review by whatston3 — APPROVED** [review]
Works alright, very clearly radioactive.

**Comment by whatston3** [comment]
Gonna merge this, works well, clearly radioactive - if it needs reversion, so be it.

**Pull request merged**